src/codemagic/tools/_app_store_connect/action_groups/apps_action_group.py

```diff
@@ -136,3 +156,121 @@ def list_app_store_versions_for_app(
+        """
+        Expire all application builds except the given build(s)
+        """
+        builds_to_skip: Set[ResourceId] = set()
+
+        if isinstance(excluded_build_id, ResourceId):
+            builds_to_skip.add(excluded_build_id)
+        elif excluded_build_id is not None:
+            builds_to_skip.update(excluded_build_id)
+
+        builds = self.list_builds(application_id=application_id, not_expired=True, should_print=should_print)
+        return [
+            self.expire_build(
+                build_id=build.id,
+            )
+            for build in builds
+            if build.id not in builds_to_skip
+        ]
+
+    @cli.action(
+        'expire-build-submitted-for-review',
+        AppArgument.APPLICATION_ID_RESOURCE_ID,
+        action_group=AppStoreConnectActionGroup.BUILDS,
+    )
+    def expire_build_submitted_for_review(
+        self,
+        application_id: ResourceId,
+        should_print: bool = False,
+    ) -> Optional[Build]:
+        """
+        Expire application build that is currently waiting for review, or is currently in review
+        """
+
+        states_to_cancel = (
+            BetaReviewState.WAITING_FOR_REVIEW,
+            BetaReviewState.IN_REVIEW,
+        )
+
+        builds = self.list_builds(
+            application_id=application_id,
+            not_expired=True,
+            beta_review_state=states_to_cancel,
+            should_print=should_print,
+        )
+        try:
+            return self.expire_build(builds[0].id)
+        except IndexError:
+            return None
+
+    @cli.action(
+        'cancel-review-submissions',
+        AppArgument.APPLICATION_ID_RESOURCE_ID,
+        AppStoreVersionArgument.PLATFORM,
+        ReviewSubmissionArgument.REVIEW_SUBMISSION_STATE,
+        action_group=AppStoreConnectActionGroup.APPS,
+    )
+    def cancel_review_submissions(
+        self,
+        application_id: ResourceId,
+        platform: Optional[Platform] = None,
+        review_submission_state: Optional[Union[ReviewSubmissionState, Sequence[ReviewSubmissionState]]] = None,
+        should_print: bool = False,
+    ) -> List[ReviewSubmission]:
+        """
+        Find and cancel review submissions in App Store Connect for the given application
+        """
+
+        review_submissions = self.list_review_submissions(
+            application_id,
+            platform,
+            review_submission_state,
+            should_print,
+        )
+
+        return [self.cancel_review_submission(submission.id) for submission in review_submissions]
+
+    @cli.action(
+        'list-review-submissions',
+        AppArgument.APPLICATION_ID_RESOURCE_ID,
+        AppStoreVersionArgument.PLATFORM,
+        ReviewSubmissionArgument.REVIEW_SUBMISSION_STATE,
+        action_group=AppStoreConnectActionGroup.REVIEW_SUBMISSIONS,
+    )
+    def list_review_submissions(
+        self,
+        application_id: ResourceId,
+        platform: Optional[Platform] = None,
```

Comment:
As with `cancel_review_submissions` above. Either use `AppStoreVersionArgument.PLATFORM_OPTIONAL` in action declaration, or define `platform` argument with proper default value.